### PR TITLE
JUCX: get/put nbi functionality.

### PR DESCRIPTION
**Pull request by petro-rudenko**
## What
JUCX: get/put nbi functionality.

## Why ?
Needed for the cases when a callback is not needed, to reduce callback overhead.

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12194/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 10-Sep-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/7de8d597494e630033f97405fa6856ddb7b86f6a/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/bcce9451ffa13c950abc598a3c56bebfc5f5075f/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/4db7ca0d88bd605c35cc2624c0090754e942c6bf/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/412183ef3b42a5da0e5ee5ecdee7152ba7f40a1a/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/1e915281c1e9db38bc66aa2124956bd24e619fc7/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/66bcfa0af146e29856df94aba016cc0e80eb234f/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/18dab931e09c48a3f93200e4b196f099725975c2/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/9ed8b8029545c16124d9bc4290bd7af8e259bc7f/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/8d37cb383cbb9748566849afbed8e8c156b17920/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/146a545a18e856bb50f63116cbcc102bc24939fa/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/e51aab7cd9f4f9da550e00ede239732b785c7789/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/0d4b1e21e3ba4c04a0334f38fd674b3d55efc98f/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/b7c90a036a52207d36d08de377871b305a0e4b17/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/e749b1616459fcfbaf30664932232ef1d117f46a/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/aa9a9423c4e9ef45aaa48273776375a63d4a746a/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/8ea11419f459d56f03e4b79ca2255f17cf5cf2bd/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/d5144683d5a7e0b870b2bd28c3d17810acc3599d/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/f518928cfa27892e69588cfca03bf611ea767ccc/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/f5c65b9afff83c2cdcbceeb4be4631ab80deeb8c/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/4a81723678fb41aa55442b0de006f5a83c409b4b/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/1ff43640e6eb307ddf73141c7b56aa4abb994dfa/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/5ad0960faaeac4a5f9b0c69fd331606727b915d9/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/1b1e7902a1223964e8acb6e5513065fe4f1d41a5/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/ecfd192f91df086e70ff0f855eddbc63661b07df/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/5c29372ac68f3df30c64a1e3ac7389c4f2285a2f/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/c2ce435b77e1d9d67b7a7bee3ee2c0364d13f0d6/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/a99ef6392a329d95ca0bc47542abda8934aa217e/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/ba1e2dd8920361aefed5cfd70f89ab8db125f22e/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/5926c380107eac63047b7abaee0362a5/raw/eff85808167202206963bd431a08139af7e05b99/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

File: bindings/java/src/main/native/endpoint.cc [hunk 1]
```diff
@@ -123,6 +123,20 @@ Java_org_openucx_jucx_ucp_UcpEndpoint_putNonBlockingNative(JNIEnv *env, jclass c
...
+JNIEXPORT void JNICALL
+Java_org_openucx_jucx_ucp_UcpEndpoint_putNonBlockingImplicitNative(JNIEnv *env, jclass cls,
+                                                                   jlong ep_ptr, jlong laddr,
+                                                                   jlong size, jlong raddr,
+                                                                   jlong rkey_ptr)
+{
+    ucs_status_t status = ucp_put_nbi((ucp_ep_h)ep_ptr, (void *)laddr, size, raddr,
+                                      (ucp_rkey_h)rkey_ptr);
+
+    if ((status != UCS_OK) && (status != UCS_INPROGRESS)) {
```
Comment (dmitrygx) [inline bindings/java/src/main/native/endpoint.cc]: can do `if (UCS_STATUS_IS_ERR(status)) { ...`

File: bindings/java/src/main/native/endpoint.cc [hunk 2]
```diff
@@ -137,6 +151,20 @@ Java_org_openucx_jucx_ucp_UcpEndpoint_getNonBlockingNative(JNIEnv *env, jclass c
...
+JNIEXPORT void JNICALL
+Java_org_openucx_jucx_ucp_UcpEndpoint_getNonBlockingImplicitNative(JNIEnv *env, jclass cls,
+                                                                   jlong ep_ptr, jlong raddr,
+                                                                   jlong rkey_ptr, jlong laddr,
+                                                                   jlong size)
+{
+    ucs_status_t status = ucp_get_nbi((ucp_ep_h)ep_ptr, (void *)laddr, size, raddr,
+                                      (ucp_rkey_h)rkey_ptr);
+
+    if ((status != UCS_OK) && (status != UCS_INPROGRESS)) {
```
Comment (dmitrygx) [inline bindings/java/src/main/native/endpoint.cc]: can do `if (UCS_STATUS_IS_ERR(status)) { ...`

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12210/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 2 of 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 11-Sep-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/ea8c2c1087d67307983b1493d35af12b814acd14/_main) | :x: FAILURE
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/636be2fb30c5fd9b69277b11f128a25336d89034/hpc-test-node-legacy_W1) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/6d64040bccb3a67be76974eb9bc36701016b58b0/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/9cf03857570f5ca6b0b31c85e628d64a7feb5fa4/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/5045f2688ed1e2ca4ccf3ff8395f2bb88a0e1f41/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/705fab9a357102d2c7abe890bff4c016da571bfe/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/dff53692563ee1c6a40e40aade58bacdb9900e07/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/cd8aa0b72912983626ea603ccca0218ca75920d0/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/97537a44fe9a97d6ac477c875987fb208cccb89b/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/89938eda430c15e8fdbc781c8d79dbafa8f47990/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/834d9cce86eb122e723db9e5b414d9dccdbde393/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/8bc175879f662aecfdfbf7d85ee5bc21db3de6ad/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/56845d21eff2430bdba8e14c2b67eae3dd291f0b/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/f3ac0148aa13e01444eb50c094978435f480292f/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/13074f50b130a314167395d1c742ee95d4569fcd/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/b8fa657c5b9cdfc77ef46f66509f9f90776ab575/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/9bb6801c596d7d558e7e645c3e57d7e0dcb85e55/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/59b5942107e95ec9c4ce71096f5a39a7322600a2/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/50763d84ca2c79d350ad94ca07f4aade30a447e2/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/8c9a8ff99240eb9abcf77e295bc5e1dc5fc78200/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/7965a60418eed639b742d3953c233fc4115c42e5/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/026f59db11541e53a7b4fb041a4211ae48eaefde/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/f46621a842a7fbd174edae7a186f9e8f373a959a/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/7893f703e5489dbf16cd44cfd10bca33b2706af0/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/766596f16556176360f61f704ad4ad45c491d7ef/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/8114c617cdff7661b9ae7aff26453d54b7775ecf/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/fdd7c45c6978b015acc52b4c8c26e0512a4cfa9f/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/a3a847f05f21b9fb23c90a2f5d6324b312d2b8e3/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/672650ee48abf20e0a9905f78e2b7366/raw/e126af770ee3ff5bc22e1bf2295abe2abb80d50d/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by yosefe** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12223/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 11-Sep-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/727cb684288dc2579568ec4354622abfd2e88c05/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/d37ed9981447f38572f6fa3247669f3289769909/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/0d7067bdc2681712a3c42130c85d93f65e97e4d5/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/ff26588cc58d2604a23c86e5459a51fb59c3e24a/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/9e9ff7d3e18c149c8a0a3c987e9a8bca41ac91c0/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/4e278c2d14974b3871f5bba3d7b93993d5e9fcf8/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/f52bda3c3d73f45d0deac0088a5f5caa6bb40e87/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/c7e69fb34b9a943d71a351d9aca52b07dad47728/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/d16a425797982b76ae43f427aab138bd73054ea7/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/7e1f4499e1129cdfaeeb03dd2efc8281782a78f3/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/758b5fd72c7e3831338d51e317850295e4d5a3a4/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/4a96c00ca4e84c14d2bcbf415645589a7c9424bf/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/684a61855e1413bf71247be3bc8b29401c4d5a03/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/6f63e6642c2104739f9d10c93aef1c779f50a83e/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/9b7bdde4643ba1d7b08120c42daf751cbcb369e4/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/a671571dd527b0ae256c2c15796ffe75fccb81f8/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/52a6e889a96e04432803547ac5727546c8644493/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/fb4ffa3f6a6ef315abe778dcfdfc3dcc14ba85f7/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/f0247d99feae246e9cd44b77d3bdfe583db0cad7/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/cd027d605b8946a31cbd1111163e1277c50c92ac/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/54b0f063f10a309423a55aa913b2bf3978ed2ba3/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/99504e653c3e174f790dd333f954a9c436297cb7/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/32936d7f7833e7b00c54754b7d4e942dad3cb442/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/0a79a789155374cc9ea5b87689b998b094096e70/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/9e13682c7b07704eb5aad1fb86b0c1866434da69/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/bc0db8e71477971943cc83a3afe61fb19a913ee9/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/282cd68c397751a03b46822e5754c83be4266e6d/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/581bef53c3f237f9c66e53a72945b17887f466fd/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/407c36b0e96ff78e143ddeee2ef55a4b/raw/97d91df89531c65ecae17065483c9339d8b57408/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Pull request merged**